Library/Homebrew/formula.rb

```diff
@@ -1528,9 +1528,9 @@ def std_cmake_args(install_prefix: prefix, install_libdir: "lib", find_framework
   end
 
   # Standard parameters for Go builds.
-  sig { params(ldflags: T.nilable(String)).returns(T::Array[String]) }
-  def std_go_args(ldflags: nil)
-    args = ["-trimpath", "-o=#{bin/name}"]
+  sig { params(output: T.any(String, Pathname), ldflags: T.nilable(String)).returns(T::Array[String]) }
+  def std_go_args(output: bin/name, ldflags: nil)
```

Comment:
Can we also make `ldflags` take an `Array` so we can drop the `ldflags: ldflags.join(" ")` pattern?